content/departments/product-engineering/engineering/process/releases/index.md

```diff
@@ -120,6 +116,117 @@ The release captain has unlimited power to make changes to the release branch to
 
 Most issues are non-blocking. Fixes to non-blocking issues can be fixed in `main` by the code owner who can then `git cherry-pick` those commits into the release branch with the approval of the release captain. Alternatively, broken features can be reverted out of the release branch or disabled via feature flags if they aren't ready or are too buggy.
 
+## Patch release process
+
+> As of Jan 31, 2022, the Delivery team still owns the release process hence the role of Release Captain, `us` in the process below is always refering to the Delivery team. However, this may change in the future when we transit to a rotation base Release Captain.
+
+### 1) Kickstart a patch release
+
+CE or the products team will start requesting a patch release by submitting a [patch release request]. The patch release to [revert poor onboarding UX change] is a good example.
+
+Upon the Delivery team or the Release Captain receives the patch release request and we have decided to rollout a new patch release, the Release Captain should follow the instruction in the [patch release request] issue to kickstart the patch release process.
+
+The instruction is located at the bottom of the [patch release request]. At a high level, it contains the following
+
+- Open a PR to update [release-config.jsonc] config file with the upcoming release information and merge the PR
+- Utilize the [Sourcegraph release tool] to initialize the next step
+
+### 2) Patch release tracking issue
+
+The [Sourcegraph release tool] creates a patch release tracking issue which contains a list of action items the Release Captain has to perform. [3.36.2 patch release tracking issue] is a good example.
+
+The Release Captain should review and follow the instruction in the patch release tracking release for next steps. At a high level, it contains the followin steps
+
+#### 2.1) Cherry-pick patch commits from `main` into the release branch
```

Comment:
I wonder how valuable it is to include these steps - they're already included in the tracking issue itself, and it means that if we change that issue this list can get out of sync.

Maybe instead, we can just summarize the intent of the process? Build a release candidate, verify it passes CI, publish final image and update documentation/references.

Reply:
+1 - the release issue documents this, and the issue template is already available in the handbook